ZeCamp/ZeCamp/AppDelegate.swift

```diff
@@ -10,6 +10,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window.makeKeyAndVisible()
         self.window = window
         window.rootViewController = UIViewController()
+        
+        let scheduleUrl = Bundle.main.bundleURL.appendingPathComponent("MyResources").appendingPathComponent("schedule.json")
```

Comment:
You can also use `Bundle.main.url(forResource: "schedule", withExtension: "json", subdirectory: "MyResources")`.

Other than typing fewer characters, it has a couple of advantages:
• It actually confirms whether the resource exists, returning `nil` otherwise.
• It handles localisation of the resource.